ResearchStack2UI/ResearchStack2UI/Styleguide/UIColor+BridgeKeyNames.swift

```diff
@@ -35,65 +35,146 @@ import UIKit
+    /// highlighted.
+    ///
+    /// Default = `secondaryTintColor.withSaturationMultiplier(1.2)`.
+    @objc open class var darkSecondaryTintColor: UIColor {
+        return secondaryTintColor.withSaturationMultiplier(1.2)
+    }
+    
+    
+    /// MARK: App background - default colors
+    ///
+    /// - note: The backgrounds are set to the the reverse of the `usesLightStyle` property. This property is
+    /// used for *foregound* elements throughout the app to denote whether or not the element should be
+    /// displayed using a dark or light color theme. For example, on a dark blue background you would set
+    /// `usesLightStyle = true` to use the light elements for text, buttons, and progress indicators (white
+    /// by default); while on a white background you would set `usesLightStyle = false` to use the dark
+    /// elements for text, buttons, and progress indicators (dark gray or `darkPrimaryTintColor` by default).
+    
+    /// Background color for views that should have a light background.
+    ///
+    /// Default = `UIColor.white`.
+    @objc open class var appBackgroundLight: UIColor {
+        return UIColor.white
+    }
+    
+    /// Background color for views that should have a dark background.
+    ///
+    /// Default = `UIColor.primaryTintColor`
+    @objc open class var appBackgroundDark: UIColor {
+        return UIColor.primaryTintColor
+    }
+    
+    
+    /// MARK: App text - default colors
+    
+    /// Color for text throughout the app when presented on a dark background.
+    ///
+    /// Default = `UIColor.white`.
+    @objc open class var appTextLight: UIColor {
+        return UIColor.white
+    }
+    
+    /// Color for text throughout the app when presented on a light background.
+    ///
+    /// Default = `UIColor.appDarkGray`.
+    @objc open class var appTextDark: UIColor {
+        return UIColor.appDarkGray
+    }
+    
+    
+    /// MARK: Shades of gray
+    
     /// Override to set the dark gray color used throughout the app for elements on a light background.
+    ///
+    /// Default = (65, 72, 89)
     @objc open class var appDarkGray: UIColor {
         return UIColor(red: 65.0 / 255.0, green: 72.0 / 255.0, blue: 89.0 / 255.0, alpha: 1.0)
     }
     
     /// Override to set the light gray color used throughout the app for elements on a light background.
+    ///
+    /// Default = (141, 147, 161)
     @objc open class var appLightGray: UIColor {
         return UIColor(red: 141.0 / 255.0, green: 147.0 / 255.0, blue: 161.0 / 255.0, alpha: 1.0)
     }
     
     /// Override to set the very light gray color used throughout the app for elements such as line
     /// separators.
+    ///
+    /// Default = #EDEDED (237, 237, 237).
     @objc open class var appVeryLightGray: UIColor {
         return UIColor(white: 237.0 / 255.0, alpha: 1.0)
     }
     
     
-    // MARK: App background - default colors
+    /// MARK: Status colors
```

Comment:
three /'s on a MARK: comment?

Reply:
Yup, that includes them in the docs along with the explanatory comment under one of the marks.